### ISSUE TITLE
ddu@18.0.1.6 Unable to install due to "Could not find 'DDU v18.0.1.6'!"

### ISSUE THREAD

**Issue by CypherNL** [lukesampson/scoop-extras#2519]
```
Installing 'ddu' (18.0.1.6) [64bit]
DDU%20v18.0.1.6.exe (1,4 MB) [================================================================================] 1
Checking hash of DDU%20v18.0.1.6.exe ... ok.
Extracting DDU-18.0.1.6.7z ... Could not find 'DDU v18.0.1.6'! (error 16)
At C:\Users\Melvin\scoop\apps\scoop\current\lib\core.ps1:506 char:9
+         throw "Could not find '$(fname $from)'! (error $($proc.ExitCo ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find ....6'! (error 16):String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find 'DDU v18.0.1.6'! (error 16)
```

If you need more information, please let me know.

**Comment by Ash258** [comment]
Will be fixed as soon as lukesampson/scoop#3540 is in master

**Comment by Stanzilla** [comment]
Any ETA?

**Comment by rasa** [comment]
https://github.com/lukesampson/scoop/pull/3540 is in master. Are you saying the problem still exists?

**Comment by Ash258** [comment]
@rasa No it is not in master yet. https://github.com/lukesampson/scoop/compare/develop

**Comment by rasa** [comment]
@Ash258 Well duh. I saw it was merged, but didn’t notice where.

**Comment by Stanzilla** [comment]
This is still an issue, when is the next merge window?

**Comment by Stanzilla** [comment]
Still broken :(

**Comment by Ash258** [comment]
Fixed with lukesampson/scoop@544e6094cc1498319a24d082496ce495c54c562c
